daemon/src/maker_inc_connections.rs

```diff
@@ -84,14 +84,16 @@ impl Actor {
         }
     }
 
-    async fn send_to_taker(&self, taker_id: TakerId, msg: wire::MakerToTaker) -> Result<()> {
+    async fn send_to_taker(&mut self, taker_id: &TakerId, msg: wire::MakerToTaker) -> Result<()> {
         let conn = self
             .write_connections
-            .get(&taker_id)
+            .get(taker_id)
             .context("no connection to taker_id")?;
 
-        // use `.send` here to ensure we only continue once the message has been sent
-        conn.send(msg).await?;
+        if conn.send(msg).await.is_err() {
+            tracing::info!(%taker_id, "Bad connection to taker, removing ...");
```

Comment:
nit: `s/Bad/Lost` ? (a bad connection is an ambiguous term, it can often refer to a connection that is still active, but unreliable/noisy)